vm/vm.go

```diff
@@ -159,6 +162,11 @@ func New(
 		allocatedNamespaces.Add(option.Namespace)
 	}
 
+	abiString, err := codec.GetVMABIString((*actionRegistry).GetRegisteredTypes())
```

Comment:
I have doubts about placing the `abiString` initialization in `vm.New`, but it throws an error and takes some time (less than 1 ms). I don't have a better alternative, though.